docs_user/modules/openstack-edpm_adoption.adoc

```diff
@@ -392,6 +430,31 @@ endif::[]
 EOF
 ----
 
+* Optionally enable neutron-sriov-nic-agent in the OpenStackDataPlaneNodeSet CR
++
+[source,yaml]
+----
+oc patch openstackdataplanenodeset openstack --type='json' --patch='[
+  {
+    "op": "add",
+    "path": "/spec/services/-",
+    "value": "neutron-sriov"
+  }, {
+    "op": "add",
+    "path": "/spec/nodeTemplate/ansible/ansibleVars/edpm_neutron_sriov_agent_SRIOV_NIC_physical_device_mappings",
+    "value": "dummy_sriov_net:dummy-dev"
+  }, {
+    "op": "add",
+    "path": "/spec/nodeTemplate/ansible/ansibleVars/edpm_neutron_sriov_agent_SRIOV_NIC_resource_provider_bandwidths",
+    "value": ""
```

Comment:
it would be helpful to see an actual non-empty value to copy (above and here). (Also below.)

Reply:
done